changelog.json

```diff
@@ -1,5 +1,20 @@
 {
   "releases": {
+    "1.6.3": [
+      "[Added] Provide option to discard when files would be overwritten by a checkout - #6755. Thanks @mathieudutour!",
```

Comment:
```suggestion
      "[Added] Option to discard when files would be overwritten by a checkout - #6755. Thanks @mathieudutour!",
```

Reply:
Are we still shipping this @billygriffin. We discussed not adding this in our stashing meeting. Also, my bad for just now bringing it up, seeing the release note jogged my memory.

Reply:
Unshipped in #7016 which is targeting this branch